docs/arf.md

```diff
@@ -1041,26 +1041,26 @@ as follows:
     V of the Regulation.
 
 -   **Electronic attestation of attributes issued by or on behalf of a
-    public sector body responsible for an authentic source (PuB-EAA)**:
+    public sector body responsible for an Authentic Source (PuB-EAA)**:
     An electronic attestation of attributes issued by a public sector
-    body that is responsible for an authentic source or by a public
+    body that is responsible for an Authentic Source or by a public
     sector body that is designated by the Member State to issue such
     attestations of attributes on behalf of the public sector bodies
-    responsible for authentic sources in accordance with Article 45f and
+    responsible for Authentic Sources in accordance with Article 45f and
     with Annex VII of the Regulation.
 
 -   **Non-Qualified EAA:** An EAA which is not QEAA or PuB-EAA.
 
 Please note that the differences between these types of attestation are
 purely legal. For example, a diploma may be a QEAA or a non-qualified
 EAA, depending on whether it is issued by a qualified trust service
-provider (QTSP) or by an unqualified one. Similarly, an mDL may be
+Provider (QTSP) or by an unqualified one. Similarly, an mDL may be
 issued as a PuB-EAA, a QEAA, or a non-qualified EAA, depending on the
 legal status of the party issuing mobile driving licenses in each Member
-State, in addition to being an authorised mDL Provider in accordance
+State, in addition to being an authorized mDL Provider in accordance
```

Comment:
We should keep the British spelling `authorised`

Reply:
Updated in accordance